python/cudnn/gemm/cutedsl/dense/swiglu/jax_api.py

```
@@ -0,0 +1,120 @@
+    Returns a plain ``(ab12_tensor, c_tensor)`` tuple of fresh n-major JAX arrays.
+    ``alpha`` is a static (trace-time) parameter.
+
+    Supports the non-quantized kernel only: the quantized kernel's compiled signature
+    carries None-typed parameters the XLA FFI bridge cannot supply -- use
+    ``gemm_swiglu_wrapper_sm100`` (eager) for blockscaled MXFP8 inputs from JAX.
+    """
+    ab12_dtype = _convert_to_cutlass_data_type(ab12_dtype)
+    c_dtype = _convert_to_cutlass_data_type(c_dtype)
+    acc_dtype = _convert_to_cutlass_data_type(acc_dtype)
+
+    m, _, l = a_tensor.shape
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)